src/kOS/Screen/TermWindow.cs

```diff
@@ -259,6 +223,26 @@ public override void Open()
             base.Open();
             BringToFront();
             guiTerminalBeepsPending = 0; // Closing and opening the window will wipe pending beeps from the beep queue.
+
+            GetFontIfChanged();
+        }
+
+        private void GetFontIfChanged()
+        {
+            int newSize = shared.Screen.CharacterPixelHeight;
+            string newName =  SafeHouse.Config.TerminalFontName;
+            if (fontSize != newSize || !(tryFontNames[0].Equals(newName)))
+            {
+                fontSize = newSize;
+                tryFontNames[0] = newName;
+                font = AssetManager.Instance.GetSystemFontByNameAndSize(tryFontNames, fontSize, false);
+
+                terminalLetterSkin.label.font = font;
+                terminalLetterSkin.label.fontSize = fontSize;
+
+                // TODO - There will probably be other measurements that have to get recalculated
+                // here too.
```

Comment:
I'm removing this todo, it doesn't look like there is anything that needs to be added here for now.